src/Nethermind/Nethermind.Blockchain.Test/Validators/TxValidatorTests.cs

```diff
@@ -243,5 +243,28 @@ public void Transaction_with_init_code_above_max_value_is_rejected_when_eip3860E
             TxValidator txValidator = new(1);
             txValidator.IsWellFormed(tx, releaseSpec).Should().Be(expectedResult);
         }
+
+        [TestCase(TxType.EIP1559, false, ExpectedResult = true)]
+        [TestCase(TxType.Blob, false, ExpectedResult = false)]
+        [TestCase(TxType.EIP1559, true, ExpectedResult = false)]
+        [TestCase(TxType.Blob, true, ExpectedResult = true)]
```

Comment:
You could generate those test with Source for all TxTypes

Reply:
Too complicated to do with sources, the current state is nicely simple to read